core/protocols/aws-json-protocol/src/main/java/software/amazon/awssdk/protocols/json/BaseAwsJsonProtocolFactory.java

```diff
@@ -181,13 +183,6 @@ protected JsonContentTypeResolver getContentTypeResolver() {
         return AWS_JSON;
     }
 
-    /**
-     * @return Default JsonNode value factory.
-     */
-    protected JsonValueNodeFactory getJsonValueNodeFactory() {
```

Comment:
Introduced in the previous PR, it can be removed now that we carry this in the builder and we don't need to expose it here.